cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 1/2/2024 (PENDING)_
 **Bugfixes:**
 
 - Now 'node_modules' will not be ignored if a project path or a provided path to spec files contains it. Fixes [#23616](https://github.com/cypress-io/cypress/issues/23616).
+- Updated display of assertions to escape markdown formatting and properly render bold styles for assertion values. Fixes [#24960](https://github.com/cypress-io/cypress/issues/24960) and [#5679](https://github.com/cypress-io/cypress/issues/5679).
```

Comment:
```suggestion
- Updated display of assertions to escape markdown formatting and properly render bold styles for assertion values. FFixes [#24960](https://github.com/cypress-io/cypress/issues/24960) and [#28100](https://github.com/cypress-io/cypress/issues/28100).
```

Reply:
Addressed in [`a28e023` (#28583)](https://github.com/cypress-io/cypress/pull/28583/commits/a28e0235737a38393abc26f21ce58e35f941a49f)